plugins/Referrers/API.php

```diff
@@ -70,7 +70,7 @@ public function get($idSite, $period, $date, $segment = false, $columns = false)
             $table->setMetadata(DataTable::EXTRA_PROCESSED_METRICS_METADATA_NAME, $processedMetrics);
         });
 
-        if (!empty($requestedColumns)) {
+        if (!empty($columns)) {
```

Comment:
Looks like this has been a typo for years. At that point `$requestedColumns` does not exist, so at the moment the below code was never executed. Based on the code logic, it seems this was meant to be a check for `$columns`, as this can be provided as API parameter.

Reply:
Moving this to it's own PR, as it seems to have a bigger impact.